test/teos/e2e/teos.conf

```diff
@@ -11,6 +11,10 @@ btc_feed_connect = localhost
 btc_feed_port = 28332
 
 [teos]
+api_bind = localhost
```

Comment:
This are the exact same as the defaults, so I don't think they are actually necessary.  In fact, some of the other may also be taken off.

Reply:
Removed api_bind, api_port, rpc_bind, rpc_port, and all the btc_feed_* settings.